docs/zenodo.rst

```diff
@@ -321,6 +321,30 @@ users with access to your account can see their files.
     directory and upload the archive to Zenodo Sandbox, then download and unzip
     it the next time it is needed.
 
+    You will also need to update the rule for any scripts that use the output
+    from this directory to take the rule ouput as an input.
+
+    .. code-block:: python
+
+        rule figure:
+            input:
+                rules.simulation.output
+            output:
+                "src/figures/figure.pdf"
+            script:
+                "src/scripts/figure.py"
+
+    You will also need to update the ``showyourwork.yml`` config file to reflect
+    the new dependency.
```

Comment:
I'd probably remove this next block and just say something like "don't forget to update your `showyourwork.yml`", since this syntax is already discussed above.

Reply:
I've found that you need to explicitly point to the *directory* rather than individual files if using the `directory(...)` output. This section only shows dependencies on individual files, which won't work from what I have found, so I think this might actually be valuable to detail explicitly? It might be obvious, but definitely caused me some trouble as someone who was coming with zero experience of these different systems.